pkg/build/controller/controller.go

```diff
@@ -220,7 +223,7 @@ func (bc *BuildPodController) CancelBuild(build *buildapi.Build, pod *kapi.Pod)
 		return err
 	}
 
-	glog.V(2).Infof("Build %s was successfully cancelled.", build.Name)
+	glog.V(2).Infof("Build %s/%s was successfully cancelled.", build.Namespace, build.Name)
```

Comment:
----- Original Message -----
> > @@ -220,7 +223,7 @@ func (bc *BuildPodController) CancelBuild(build
> > *buildapi.Build, pod *kapi.Pod)
> >  		return err
> >  	}
> >  
> > -	glog.V(2).Infof("Build %s was successfully cancelled.", build.Name)
> > +	glog.V(2).Infof("Build %s/%s was successfully cancelled.",
> > build.Namespace, build.Name)
> 
> @smarterclayton if it is so, then I dont thing logs like:
> - Updating build status ...

Not that important - 4

> - Build 1234 has been generated from 4321 build configuration

Not that important - 4

> - Error resolving image tag reference

Important, 2 

> - Failed to parse request URL

Important, 2

> - Failed to extract information from web hook

Important, 2

> - ....
> should be in the Hail Mary group
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/2452/files#r31056800